website/docs/explorer/querying.mdx

```diff
@@ -59,84 +59,23 @@ The schema looks like
 |------------| ---------| ----------------|---------|----------|------------------------|
 |`default/progress-delivery-demo2-32` | `flux-system`   | `HelmRelease` | `podinfo` | `Success` | `Release reconciliation succeeded`  |
 
-And represented in the UI like
+You can open the query filter settings by clicking on the filter button:
 
-![explorer](imgs/explorer-query-overview.png)
+![explorer](imgs/explorer-filter-open.png)
```

Comment:
We could reduce this screenshoot (may be just showing two categories ) ... we could have the same info with less space.